resources/sass/app.scss

```diff
@@ -110,7 +109,7 @@ li {
 .store > header {
     flex-direction: row;
 
-    @media (max-width: $width-breakpoint-tablet) {
+    @media (max-width: $max-width-breakpoint-tablet) {
```

Comment:
```suggestion
    @media (max-width: $max-width-breakpoint-mobile) {
```

Reply:
my bad. this should actually be `$max-width-breakpoint-mobile` in every case

Reply:
For git integrity purposes it was easier for me to push these changes myself than to commit your suggestions. The right breakpoints should be included now though.

Reply:
Thanks! No worries about that, wasn't sure it those would be useful.